bundles/org.openhab.binding.astro/src/main/java/org/openhab/binding/astro/internal/action/AstroActions.java

```diff
@@ -61,7 +59,7 @@ public void setThingHandler(@Nullable ThingHandler handler) {
     }
 
     @RuleAction(label = "get the azimuth", description = "Get the azimuth for a given time.")
-    public @Nullable @ActionOutput(name = "getAzimuth", label = "Azimuth", type = "org.openhab.core.library.types.QuantityType<javax.measure.quantity.Angle>") QuantityType<Angle> getAzimuth(
+    public @Nullable @ActionOutput(name = "getAzimuth", label = "Azimuth", type = "org.openhab.core.library.types.QuantityType<javax.measure.quantity.Angle>") QuantityType<?> getAzimuth(
```

Comment:
You're correct, I made this modifications because the IDE complained with : 
`Unsafe null type conversion (type annotations): The value of type '@NonNull QuantityType<@NonNull Angle>' is made accessible using the less-annotated type 'QuantityType<Angle>'`
...but in reality the underlaying cause comes from not @NonNullByDefault annotated classes. I will revert these, as I have a plan to increase non null annotations in coming PR (as well as removing Calendar usage later).